### PR TITLE
Added user guide for bianca

### PR DESCRIPTION
**Pull request by dbampalikis**

### CODE REVIEW

File: user-guide-bianca.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+In another terminal
+* Login to bianca running:
+```bash
+ssh -A <username>-<project_name>@bianca.uppmax.uu.se
+```
+* Copy image files from `/proj/<project_name>/nobackup/wharf/<username>/<username>-<project_name>` to the folder you want to store the images using the cp command and replacing to the correct locations:
+```bash 
+cp cp /proj/<project_name>/nobackup/wharf/<username>/<username>-<project_name>/ega-crypt4gh_crypt4gh-1.0.0.sif .
+cp cp /proj/<project_name>/nobackup/wharf/<username>/<username>-<project_name>/ega-crypt4gh_crypt4gh-keygen-1.0.0.sif .
```
Comment (pontus) [inline user-guide-bianca.md]: Double `cp`. And is there any upside of copying these before usage?

File: user-guide-bianca.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+```
+Running the `ls` command in this folder, you should be able to see the two `.sif` files.
+
+## Key pair generation
+In order to encrypt the file(s) with crypt4gh, you need both a personal key pair and the EGA-SE public encryption key.
+### Create a personal key pair
+The creation of a key pair is very simple using crypt4gh. Run the following command, in the folder containing the `.sif` files, replacing [my-key] with the name of the key (can be anything) and specifying a passphrase when requested:
+```bash
+singularity run ega-crypt4gh_crypt4gh-keygen-1.0.0.sif --pk <my-key>.pub --sk <my-key>.sec
+```
```
Comment (pontus) [inline user-guide-bianca.md:61]: I do not have complete overlook here, is there any benefit for them of keeping track of the crypt4gh key (as in "keys are registered to an account and output files will only be encrypted to that key").

From what I know, they need a key to be able to use crypt4gh, but there is no real need for them to save it (and when I try on my system now, it seems to behave as I remembered it should (running fine without a precreated key).

File: user-guide-bianca.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+```bash 
+mount_wharf <project_name>
+```
+where [project_name] in the form of sens20XXX.
+* Copy image files using the cp command and replacing to the correct locations:
+```bash 
+cp ega-crypt4gh_crypt4gh-1.0.0.sif <project_name>/
+cp ega-crypt4gh_crypt4gh-keygen-1.0.0.sif <project_name>/
+```
+
```
Comment (pontus) [inline user-guide-bianca.md:40]: New heading here?

And for the transfer, need to bring in the ega-se key?

**Review by pontus — CHANGES_REQUESTED** [review]

File: user-guide-bianca.md (hunk 2 above)
Comment (dbampalikis) [reply user-guide-bianca.md:61]: I think you are right on this one. I am not sure if there is a reason for them to keep track of the key. Maybe @viklund has a saying

File: user-guide-bianca.md (hunk 3 above)
Comment (dbampalikis) [reply user-guide-bianca.md:40]: Yes, that's on one of the next sections

**Review by pontus — APPROVED** [review]

**Pull request merged**